tests/Unit/DependencyInjection/Compiler/RegisterRouteEnhancersPassTest.php

```diff
@@ -30,7 +30,7 @@ public function testRouteEnhancerPass()
 
         $builder = $this->createMock(ContainerBuilder::class);
         $definition = new Definition('router');
-        $builder->expects($this->at(0))
+        $builder->expects($this->atLeastOnce())
```

Comment:
weird that we had this `at(0)` here. do the tests succeed with `$this->once()`? that would seem more accurate

Reply:
I still think it tests `\Symfony\Cmf\Component\Routing\DependencyInjection\Compiler\RegisterRouteEnhancersPass::process()` which must call `hasDefinition()` at least once

Also after that call few others could be executed which may call it, so the method could be executed more then once

Reply:
And yes tests pass with `once()`

Reply:
the old code would have returned null on futher calls. but its not that important as the implementation may call it as often as it wants.

Reply:
OTOH making it ``once()` is more strict and could be better, but I've no idea which is proffered

Reply:
i think that actually this is better. otherwise we test implementation details rather than the behaviour.